.terrahub.yml

```diff
@@ -14,15 +14,24 @@ terraform:
 template:
   provider:
     - aws:
-        region: '${var.region}'
+        region: var.region
         allowed_account_ids:
-          - '${var.account_id}'
+          - var.account_id
     - aws:
         alias: default
-        region: '${var.region}'
+        region: var.region
         allowed_account_ids:
-          - '${var.account_id}'
+          - var.account_id
+    - aws:
+        alias: sample
+        region: var.sample_region
+        assume_role:
+          - role_arn: >-
+              arn:aws:iam::var.sample_account_id:role/OrganizationAccountAccessRole
+            session_name: var.sample_account_id
   tfvars:
     account_id: 123456789012
     region: us-east-1
     tfstate_path: /tmp/.terrahub/tfstate_local/terraform-aws-landing-zone
+    sample_account_id: 987654321098
```

Comment:
Please change this account id into `123456789012`

Reply:
ignore this. it is expected a different value, to show a different account. closing it now as resolved.